Resources/Audio/Voice/Cluwne/attributions.yml

```diff
@@ -0,0 +1,12 @@
+- files: ["sound_voice_cluwnelaugh1.ogg"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "cluwne laugh 1 uploaded by ZeWaka and taken from goonstation github"
+  source: "https://github.com/goonstation/goonstation/blob/master/sound/voice/cluwnelaugh1.ogg"
+- files: ["sound_voice_cluwnelaugh2.ogg"]
+  license: "CC-BY-NC-SA-3.0"
```

Comment:
place a new line between `files` entries (same to the rest of the file)

Reply:
This should be resolved